UPGRADE.rst

```diff
@@ -52,53 +52,165 @@ returned by the Client-Server API:
 Upgrading to v1.4.0
 ===================
 
-Config options
---------------
-
-**Note: Registration by email address or phone number will not work in this release unless
-some config options are changed from their defaults.**
-
-This is due to Synapse v1.4.0 now defaulting to sending registration and password reset tokens
-itself. This is for security reasons as well as putting less reliance on identity servers.
-However, currently Synapse only supports sending emails, and does not have support for
-phone-based password reset or account registration. If Synapse is configured to handle these on
-its own, phone-based password resets and registration will be disabled. For Synapse to send
-emails, the ``email`` block of the config must be filled out. If not, then password resets and
-registration via email will be disabled entirely.
-
-This release also deprecates the ``email.trust_identity_server_for_password_resets`` option and
-replaces it with the ``account_threepid_delegates`` dictionary. This option defines whether the
-homeserver should delegate an external server (typically an `identity server
-<https://matrix.org/docs/spec/identity_service/r0.2.1>`_) to handle sending password reset or
-registration messages via email and SMS.
+**Note: As of this release, users will be unable to add phone numbers or email
+addresses to their accounts, without changes to the Synapse configuration. This
+includes adding an email address during registration.**
+
+It is possible for a user to associate an email address or phone number
+with their account, for a number of reasons:
+ * for use when logging in, as an alternative to the user id.
+ * in the case of email, as an alternative contact to help with account recovery.
+ * in the case of email, to receive notifications of missed messages.
+
+Before an an email address or phone number can be added to a user's account, 
+or before such an address is used to carry out a password-reset, Synapse must 
+confirm the operation with the owner of the email address or phone number.
+It does this by sending an email or text giving the user a link or token to confirm 
+receipt. This process is known as '3pid verification'.
+
+Previous versions of Synapse delegated this task to an
+identity server by default. In most cases this server is vector.im or
+matrix.org.
+
+In Synapse 1.4.0, for security and privacy reasons, the homeserver will no
+longer delegate 3pid validation to an identity server by default and instead the
```

Comment:
probably my bad, but we should decide if we are calling it 'verification' or 'validation'. I think 'verification'?